core/utils/toolbox.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Utility functions for the toolbox and flyout.
+ * @author aschmiedt@google.com (Abby Schmiedt)
+ */
+'use strict';
+
+goog.provide('Blockly.utils.toolbox');
+
+
+/**
+ * The information needed to create a block in the toolbox.
+ * @typedef {{
+ *            kind:string,
+ *            blockxml:(string|Node)
+ *          }}
+ */
+Blockly.utils.toolbox.BlockInfo;
```

Comment:
Given the name space is already explicit enough. I think it's fair to cut out the ``Info`` part of each of these.